activities/codebase-stewardship/assessment-issue-template.md

```diff
@@ -0,0 +1,26 @@
+---
+type: Resource
+---
+
+# Template for assessment issues
+
+The text below can be used as a template for creating an issue in a repository for codebase where we are [starting an assessment](open-assessment.md). For easiest copy-paste it is rendered with code style here.
+
+## Template text
+
+```
+**Purpose**
+
+This issue helps tracking the assessment of this repository on it's way to [stewardship](https://about.publiccode.net/activities/codebase-stewardship/) by the Foundation for Public Code. The assessment is compiled into this [spreadsheet](linkToSpreadsheet).
```

Comment:
Personally I feel that the risk of copy-pasting this and having a link that heads to `linkToSpreadsheet` is too big and I'd add this without that link and add text at the bottom to say you should link that. Then it will at least not be broken when people only follow half of the instructions.

Doesn't block merge